sonarqube/content.md

```diff
@@ -107,7 +150,15 @@ Follow these steps for your first installation:
 	    %%IMAGE%%
 	```
 
-## Upgrade SonarQube
+## Upgrade SonarQube 7.9.X LTS to another 7.9.X LTS version
+
+Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). No specific docker operation are needed, just use the new tag.
+
+## Upgrade from SonarQube 7.9.X LTS to SonarQube 8.X and later
+
+Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). On top of that, you need to switch from volumes to host-binded folder for easier next upgrade procedure. Follow the next chapiter `Upgrade from SonarQube 8.X and later` instructions, but instead of upgrading from a backup-folder to a new folder, you will upgrade from your backup-volume to a new folder. Starting with the 8.0 version, You can move your `sonar.jdbc.username` and `sonar.jdbc.password` value to `sonar.properties` instead of passing them with the `-e` flag.
```

Comment:
```suggestion
Take a look at the [Upgrade Guide](https://docs.sonarqube.org/latest/setup/upgrading/). In addition, you need to switch from volumes to host-binded folder to make the next upgrade step easier. Follow the instructions in the next section `Upgrade from SonarQube 8.X and later` , but instead of upgrading from a backup-folder to a new folder, you will upgrade from your backup-volume to a new folder. Starting with the version 8.0, you can move your `sonar.jdbc.username` and `sonar.jdbc.password` value to `sonar.properties` instead of passing them with the `-e` flag.
```

Reply:
Is host-binded the same thing as bind-mounted folders? If so, I would use bind-mounted since that's the terminology used earlier.

Reply:
yes it is !